src/deepsparse/transformers/pipelines/text_generation.py

```diff
@@ -438,10 +438,7 @@ def process_inputs(
             inputs.sequences = repeat_inputs(
                 inputs.sequences, generation_config.num_return_sequences
             )
-            if self.engine:
-                self.engine.deterministic = False
-            if self.multitoken_engine:
-                self.multitoken_engine.deterministic = False
+            self.deterministic = False
```

Comment:
@dsikka noting that we cannot change pipeline state (persistent) due to a parameter in a single inference (temporary), should only set at the last minute when passing to token generator based on inputs